eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,10 +1,23 @@
 stages:
-  - stage: Build
-    jobs:
-      - template: ../jobs/archetype-sdk-client.yml
+  - ${{ if not(endsWith(variables['Build.DefinitionName'], ' - tests')) }}:
+    - stage: Build
+      dependsOn: []
+      condition:
+      jobs:
+        - template: ../jobs/archetype-sdk-client.yml
 
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    - stage: LiveTest
+      dependsOn: []
+      jobs:
+        - template: /eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+          parameters:
+            LiveTestSamples:
+              - Name: Storage Blobs
+                NonWindowsPath: ./build/sdk/storage/blobs/samples/blobs_client_example
```

Comment:
I believe we should be able to use the NonWindowsPath on windows as well.

Reply:
NonWindowsPath differs in where the cmake generated projects output the executables. In the case of VS projects (for windows path) the exe is generated in the `Debug` folder.

Reply:
We could potentially compose it by having path prefix and binary name parameters but we'll end up with the same number of parameters and it'd be less flexible.